static/includes/FilenameEncryption.md

```diff
@@ -0,0 +1,10 @@
+&NewLine;
+
+{{< hint type=warning title="Filename Encryption"  >}}
+The rclone project has identified known issues with **Filename Encryption** in certain configurations, such as when long file names are used. See [SSH_FX_BAD_MESSAGE when syncing files with long filename to encrypted sftp storage](https://forum.rclone.org/t/ssh-fx-bad-message-when-syncing-files-with-long-filename-to-encrypted-sftp-storage/46228).
```

Comment:
two issues here, file names is two word and all caps for SFTP
```suggestion
The rclone project has identified known issues with **Filename Encryption** in certain configurations, such as when long file names are used. See [SSH_FX_BAD_MESSAGE when syncing files with long file names to encrypted SFTP storage](https://forum.rclone.org/t/ssh-fx-bad-message-when-syncing-files-with-long-filename-to-encrypted-sftp-storage/46228).
```

Reply:
This one is just the title of the forum post, so I don't think we should edit it, even if we'd quibble with the details